src/views/statutory/PositionsList.vue

```diff
@@ -282,6 +323,17 @@ export default {
   mounted () {
     this.isLoading = true
 
+    this.axios.get(this.services['oms-core-elixir'] + '/bodies').then((response) => {
+      /* The bodies that observe elections during an Agora are:
+       * ALl commissions, SUCT, CD, all WG's
+       */
+      for (const body of response.data.data) {
+        if (body.type == 'commission' || body.type == 'working group' || body.name == 'Summer University Project' || body.name == 'Comité Directeur') {
```

Comment:
also let's remove this filter, for the reasons I've written above

Reply:
So do you then want to load all the bodies? It will make for a large amount of options in the form

Reply:
yeah, still it's not our concern to choose them. Mainly because if there's another case, we'll need to update it as well.